src/prescriptions/prescription-tab-panel.component.tsx

```diff
@@ -21,40 +21,37 @@ const PrescriptionTabPanel: React.FC<PrescriptionTabPanelProps> = ({
 }) => {
   const { t } = useTranslation();
   const config = useConfig<PharmacyConfig>();
+  const isInitialized = useRef(false);
   const { sessionLocation } = useSession();
-  const { filterLocations, isLoading: isFilterLocationsLoading } = useLocationsForFiltering(config);
+  const { locations, isLoading: isFilterLocationsLoading } = useLocations(config);
   const [searchTerm, setSearchTerm] = useState('');
   const debouncedSearchTerm = useDebounce(searchTerm, 500);
-  const [locations, setLocations] = useState<SimpleLocation[]>([]);
+  const [filterLocations, setFilterLocations] = useState<SimpleLocation[]>([]);
```

Comment:
Changed this so that locations = all locations available to choose from, and filter locations = actual loading we are actively filtering on